test/Interop/Cxx/stdlib/use-std-stack.swift

```diff
@@ -0,0 +1,80 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -cxx-interoperability-mode=swift-6)
+// RUN: %target-run-simple-swift(-I %S/Inputs -cxx-interoperability-mode=upcoming-swift)
+// RUN: %target-run-simple-swift(-I %S/Inputs -cxx-interoperability-mode=upcoming-swift -Xcc -std=c++14)
+// RUN: %target-run-simple-swift(-I %S/Inputs -cxx-interoperability-mode=upcoming-swift -Xcc -std=c++17)
+// RUN: %target-run-simple-swift(-I %S/Inputs -cxx-interoperability-mode=upcoming-swift -Xcc -std=c++20)
+
+// Also test this with a bridging header instead of the StdSet module.
```

Comment:
nit: 
```suggestion
// Also test this with a bridging header instead of the StdStack module.
```

Reply:
Yes, renaming it __popUnsafe() is more in line with the convention, but this will result in compilation failures. I am still exploring this issue. Should I change this PR to the draft type?